### DISCUSSION TITLE
Welcome to Wavin-AHC-9000-mqtt Discussions!

### DISCUSSION THREAD

**Discussion by dkjonas** [dkjonas/Wavin-AHC-9000-mqtt#30] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kentvoss67** [comment]
Hej Jonas
jeg har fulgt din super fine quide og kan når jeg sender en # ud så svarer den med Besked 0 er modtaget på heat/floorE8DB84E0481E/online kl. 21.22:
True
QoS: 0 - Retain: true
hvordan ser jeg alle thermostater mm ?
